### DISCUSSION TITLE
how i remove Security Card from API Overview page

### DISCUSSION THREAD

**Discussion by mohitky2018** [dedoc/scramble#668] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ab224ff9-fe25-4a38-b1c3-ce10eb8c59ff)
how i remove that on my base api route because on that not required any auth token

**Comment by romalytvynenko** [comment, marked as answer]
https://scramble.dedoc.co/usage/authentication#excluding-a-route-from-security-requirements
